packages/core/src/components/table/table-body-row-expandable/readme.md

```diff
@@ -1,6 +1,38 @@
 # tds-table-body-row-expended
 
+### Using the `:part` Selector
 
+The `:part` selector allows you to style elements inside a shadow DOM that have been exposed via the `part` attribute. This is particularly useful for customizing the appearance of web components.
+
+In the `tds-table-body-row-expandable` component, the expandable row is exposed using the `part` attribute:
+
+```typescript:packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx
+<tr
+  class={{
+    'tds-table__row-expand': true,
+    'tds-table__row-expand--expanded': this.isExpanded,
+  }}
+  part="expand-row"
+>
+  <td class="tds-table__cell-expand" colSpan={this.columnsNumber}>
+    <slot name="expand-row" />
+  </td>
+</tr>
+```
+
+To style this part from outside the shadow DOM, you can use the `::part` pseudo-element in your CSS:
+
+```css
+tds-table-body-row-expandable::part(expand-row) {
+  background-color: var(--tds-red-100);
+  border: 1px solid var(--tds-red-200);
+}
+```
+
```

Comment:
Might be good to add a test to cover this btw.

Reply:
Yeah, good idea to ensure it is working, on it.